tests/text/helpers.py

```diff
@@ -435,3 +435,22 @@ def run_differentiability_test(
         if metric.is_differentiable:
             # check for numerical correctness
             assert torch.autograd.gradcheck(partial(metric_functional, **metric_args), (preds[0], targets[0]))
+
+
+def skip_on_connection_issues(reason: str = "Unable to load checkpoints from HuggingFace `transformers`."):
+    """Wrapper which handles HF-related tests if they fail due to connection issues.
+
+    The tests run normally if no connection issue arises, and they're marked as skipped otherwise.
+    """
+
+    def test_decorator(function: Callable, *args: Any, **kwargs: Any) -> Optional[Callable]:
+        @wraps(function)
+        def run_test(*args: Any, **kwargs: Any) -> Optional[Any]:
+            try:
+                return function(*args, **kwargs)
+            except OSError:
+                pytest.skip(reason)
```

Comment:
I may also search for key word signaling it is internet error :rabbit:

Reply:
Maybe searching for the start of the error message as updatet [here](https://github.com/PyTorchLightning/metrics/pull/940/commits/81166b25d2232cec63940074c02252ccd7ce5fdc) may help?